app/models/invitation_manager.rb

```diff
@@ -61,12 +61,12 @@ def send_waiting_list_emails(workshop)
         WorkshopInvitationMailer.notify_waiting_list(waiting_list.invitation).deliver_now
         waiting_list.destroy
       end
+    end
 
-      if workshop.host.seats > workshop.attending_students.length
-        WaitingList.by_workshop(workshop).where_role('Student').each do |waiting_list|
-          WorkshopInvitationMailer.notify_waiting_list(waiting_list.invitation).deliver_now
-          waiting_list.destroy
-        end
+    if workshop.host.seats > workshop.attending_students.length
```

Comment:
@notapatch Rubocop is reporting a `Use a guard clause instead of wrapping the code inside a conditional expression.` offence for me on this line. Could we fix this as part of this PR?

Reply:
I think that it breaks the symmetry you have one mailer in a block and the other protected by guard clause - so you are going to wonder why it's different - after some time you would realise there isn't a difference. But we can have it both ways. I'll refactor out the blocks and make both of them guard clauses.